robot/control/Src/radio/RadioLink.cpp

```diff
@@ -83,5 +85,7 @@ bool RadioLink::receive(KickerCommand& kickerCommand,
     motionCommand.bodyWVel = static_cast<float>(control->bodyW) / rtp::ControlMessage::VELOCITY_SCALE_FACTOR;
     motionCommand.dribbler = control->dribbler;
 
+    radioConnected = true;
```

Comment:
So connected is based off receiving a packet?

Reply:
Yes. Connected over WiFi is based on having received a packet. That's probably not a good measure, though. It's probably redundant with the soccer connection error which is based on there being a steady stream of packets sent back and forth.

Idk how to detect whether the radio is simply just connected to wifi or whether this is even useful at all. Since soccer connection error could either mean radio is not connected to wifi or that soccer timed out, I kind of wanted do distinguish the two. Maybe it's not necessary